### ISSUE TITLE
Donation links?

### ISSUE THREAD

**Issue by BinaryConstruct** [florianfesti/boxes#365]
Hi, this generator is amazing and I have used it numerous times over the years. I would like to contribute, but python and photography are not really my strong suit. Do you have any donation links, preferred charities or other ways I could contribute?

**Comment by Blancanegra** [comment]
It's not the first time I've thought the same!.

**Comment by florianfesti** [comment]
Well, I am putting this off over and over again as it requires getting the legal stuff ironed out. In the mean time there is a very tiny, German only page at my local hackerspace that does have a Paypal link. But I have not tried that in quite a while.

https://www.hackerspace-bamberg.de/Boxes.py

**Comment by Blancanegra** [comment]
Thank you!
I just made a donation, so could you confirm if it works now or not!

**Comment by florianfesti** [comment]
Yes, that worked. Thanks for the generous donation and the kind words!
